language-plutus-core/src/Language/PlutusCore.hs

```diff
@@ -99,6 +99,8 @@ module Language.PlutusCore
     , plcProgram
     -- * Evaluation
     , parseRunCk
+    , runCk
+    , evaluateCk
```

Comment:
We've discussed these specific functions with Manuel:

Me:
> I do not export `runCk` from `PlutusCore`, because we will have more evaluators and I think it's nice if the user explicitly specifies which one he wants by importing the corresponding module


Manuel:
> Yes, that seems like a good design decision.

Reply:
Okay, I'll make it import the specific module.

Reply:
IMO we shouldn't export `parseRunCk` either. That's actually my fault, I'll remove it too.